src/cpp/include/openvino/genai/cache_eviction.hpp

```diff
@@ -66,6 +66,22 @@ class KVCrushConfig {
     std::size_t get_budget() const {
         return budget;
     }
+
+    std::string to_string() const {
+        static const std::unordered_map<KVCrushAnchorPointMode, std::string> kv_crush_anchor_point_mode_to_string = {
+            {KVCrushAnchorPointMode::RANDOM, "RANDOM"}
```

Comment:
Add mapping for every `KVCrushAnchorPointMode`'s values

Reply:
Updated